internal/controller/utils/common.go

```diff
@@ -8,42 +8,47 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+const (
+	StorageClassEnv  = "STORAGE_CLASS"
+	ReclaimPolicyEnv = "RECLAIM_POLICY"
```

Comment:
These can be moved to the `resources` package. We should only have things in `common` if they are used across packages (or they don't fit anywhere else)

Reply:
The variables are verified in main so I will keep this here.